app/perfunwind.cpp

```diff
@@ -653,6 +656,12 @@ void PerfUnwind::analyze(const PerfRecordSample &sample)
         }
     }
 
+    if (qEnvironmentVariableIsSet("STITCH_CALLCHAIN")) {
+        if (m_currentUnwind.firstGuessedFrame != -1)
+            m_currentUnwind.frames = stitchedCallChain();
```

Comment:
`stitchCurrentCallchain();` - operate on `m_currentUnwind.frames` directly in there